File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Concepts/VisualEffects/BorderRadius/BorderRadiusPage.vb
Namespace CompanionKit.Concepts.VisualEffects.BorderRadius

	Public Class BorderRadiusPage
        'Define RadiusAll value
        Public intRadius As Integer = 20
        'Define BorderRadius Visual Effect
        Public mobjVE As Gizmox.WebGUI.Forms.VisualEffects.BorderRadiusVisualEffect
		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

            'Set default Visual Effect for TextBox
            mobjVE = New Gizmox.WebGUI.Forms.VisualEffects.BorderRadiusVisualEffect(intRadius)
            mobjTextBox.VisualEffect = mobjVE

        End Sub

       

        ''' <summary>
        ''' Handles the Click event of the mobjMinusButton control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="System.EventArgs"/> instance containing the event data.</param>
        Private Sub mobjMinusButton_Click(sender As System.Object, e As System.EventArgs) Handles mobjMinusButton.Click
            'Decrease RadiusAll value

            If (intRadius >= 5) Then intRadius = intRadius - 5 Else intRadius = intRadius

            mobjVE.BorderRadiusAll = intRadius
            mobjTextBox.Update()
        End Sub

        ''' <summary>
        ''' Handles the Click event of the mobjPlusButton control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="System.EventArgs"/> instance containing the event data.</param>
        Private Sub mobjPlusButton_Click(sender As System.Object, e As System.EventArgs) Handles mobjPlusButton.Click
            'Increase RadiusAll value

            If (intRadius <= 70) Then intRadius = intRadius + 5 Else intRadius = intRadius

            mobjVE.BorderRadiusAll = intRadius
            mobjTextBox.Update()
        End Sub
    End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Concepts/VisualEffects/BorderRadius/BorderRadiusPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Concepts.VisualEffects.BorderRadius
{
    public partial class BorderRadiusPage : UserControl
    {
        //Define RadiusAll value
        public int intRadius=20;
        //Define BorderRadius Visual Effect
        public Gizmox.WebGUI.Forms.VisualEffects.BorderRadiusVisualEffect mobjVE;
        public BorderRadiusPage()
        {
            InitializeComponent();
            //Set default Visual Effect for TextBox
            mobjVE = new Gizmox.WebGUI.Forms.VisualEffects.BorderRadiusVisualEffect(intRadius);
            mobjTextBox.VisualEffect = mobjVE;
        }


        /// <summary>
        /// Handles the Click event of the mobjMinusButton control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void mobjMinusButton_Click(object sender, EventArgs e)
        {
            //Decrease RadiusAll value
            intRadius = intRadius >= 5 ? intRadius - 5 : intRadius;
            mobjVE.BorderRadiusAll = intRadius;
            mobjTextBox.Update();
        }

        /// <summary>
        /// Handles the Click event of the mobjPlusButton control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void mobjPlusButton_Click(object sender, EventArgs e)
        {
            //Increase RadiusAll value
            intRadius = intRadius <= 70 ? intRadius + 5 : intRadius;
            mobjVE.BorderRadiusAll = intRadius;
            mobjTextBox.Update();
        }

    }
}